transcendence-app/src/chat/chat.service.ts

```diff
@@ -234,58 +233,13 @@ export class ChatService {
     const { password } = joinChatroomDto;
     if (
       foundChatroom.password &&
-      (!password ||
-        (await Password.compare(foundChatroom.password, password)) === false)
+      (!password || !(await Password.compare(foundChatroom.password, password)))
```

Comment:
This change comes because my static code analyzer tool keeps bothering me that this is better (kind of makes sense)